hw/ip/aon_timer/doc/_index.md

```diff
@@ -20,8 +20,10 @@ See that document for an overview of how it is integrated into the top level sys
 
 The always-on wakeup timer operation is straightforward.
 A count starts at 0 and slowly ticks upwards (one tick every `N + 1` clock cycles, where `N` is the pre-scaler value).
-When it reaches / exceeds the wake count, a level wakeup signal is sent to the power manager and a level IRQ is sent to the processor.
-This wakeup signal stays high until it is explicitly acknowledged by software (software must write zero to the {{<regref "WKUP_CAUSE">}} register to clear it).
+When it reaches / exceeds the wake threshold, a level wakeup signal is sent to the power manager and a level IRQ is sent to the processor.
+This wakeup signal stays high until it is explicitly acknowledged by software.
+Software must write 0 to the {{<regref "WKUP_CAUSE">}} register to clear the wakeup and 1 to {{<regref "INTR_STATE.wkup_timer_expired">}} to clear the interrupt.
+Note that if {{<regref "WKUP_COUNT">}} is not zeroed and remains at or above the wake threshold or the wakeup timer isn't disabled the wakeup and interrupt will trigger again at the next clock tick.
```

Comment:
I'd suggest a comma here: ".. or the wakeup timer isn't disabled, the wakeup and ...". Otherwise it's a bit difficult to parse the sentence.